astropy/convolution/convolve.py

```diff
@@ -546,6 +546,12 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
     complex_dtype : numpy.complex, optional
         Which complex dtype to use.  `numpy` has a range of options, from 64 to
         256.
+    dealias: bool, optional
+        Default off. Zero-pad image to allow enable explicit dealiasing
```

Comment:
```suggestion
        Default off. Zero-pad image to enable explicit dealiasing
```
"allow" xor "enable"